app/models/risk_model.py

```diff
@@ -18,5 +17,34 @@ class RiskModel(db.Model):
 
     id = Column(Integer, primary_key=True)
     title = Column(String, nullable=False, unique=True)
-    description = Column(String, nullable=False)
     text = Column(String, nullable=False)
+
+    CASES = {
+        "NOT_SPECIFIED": {"live_nearby_river": False, "live_nearby_mountain": False},
+        "EARTH_SLIDING": {"live_nearby_river": False, "live_nearby_mountain": True},
+        "FLOOD": {"live_nearby_river": True, "live_nearby_mountain": False},
+        "FLOOD_AND_SLIDING": {"live_nearby_river": True, "live_nearby_mountain": True},
+    }
+
+    # TODO: find a better way to display the texts
```

Comment:
Maybe we should get rid of the triple strings and just display it as a long string in here, i think the triple strings creates a few "\n" inside the string and might not be what we want, but for now we can keep it like this, idk.

Reply:
I'll do it, then.